docker/qchem.dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-From pennylane/base:latest
+FROM pennylane/base:latest
```

Comment:
why this change?

Reply:
It is good practice to have all docker syntax in BOLDFACE, and is used this way throughout the other dockerfiles.